### PR TITLE
Fixed bugs Re: Feedback

### PR DESCRIPTION
**Pull request by ShashankSule**
Turns out the template.yml file was deleted from my repo. Apologies.

### CODE REVIEW

File: codes/classical/q-ary_digits/ternary_golay.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+protection: 'Corrects up to 2 errors and detects up to 4 errors'
+
+features:
+
+  rate: '6/11'
+  encoders:
+    - 'Codewords are generated from the generator matrix with 5 information trits and 6 parity check trits'
+
+realizations:
+  - 'Code used in football pools with at least one good bet: \cite{doi:10.1007/BF03025254}'
```
Comment (valbert4) [inline codes/classical/q-ary_digits/ternary_golay.yml:34]: to self: doi:10.2307/2974552,doi:10.1007/BF03025254

File: codes/classical/q-ary_digits/ternary_golay.yml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  - 'Using a Golay code to create an 11-qutrit code: \cite{manual:{Prakash, Shiroman. "Magic state distillation with the ternary Golay code." Proceedings of the Royal Society A 476.2241 (2020): 20200187.}}'
+
+relations:
+  parents:
+    - code_id: binary_linear
+    - code_id: perfect
+    - code_id: quad_residue
+# cousins:
```
Comment (valbert4) [inline codes/classical/q-ary_digits/ternary_golay.yml:42]: to self: add also covering?

**Comment by valbert4** [comment]
Thanks a lot Shashank! These were very thorough and will be up soon.

**Comment by ShashankSule** [comment]
You're welcome Victor. The class this spring was a real pleasure, I learned
a lot from it!

**Pull request merged**